lib/marked.js

```diff
@@ -27,6 +27,7 @@ var block = {
   text: /^[^\n]+/
 };
 
+block.checkbox = /^\[([ x])\] +/;
```

Comment:
According to GFM docs `[v] ` is a valid way to indicate a task is complete as well.